codegens/nodejs-axios/lib/parseRequest.js

```diff
@@ -0,0 +1,194 @@
+var _ = require('./lodash'),
+  sanitize = require('./util').sanitize;
+  // path = require('path');
+
+
+/**
+ * Parses URLEncoded body from request to axios syntax
+ *
+ * @param {Object} body URLEncoded Body
+ * @param {boolean} trim trim body option
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseURLEncodedBody (body, trim, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' qs = require(\'qs\')\n',
```

Comment:
need a semicolon after the require('qs') statement

Reply:
done